modules/new_installer.py

```diff
@@ -0,0 +1,82 @@
+def download(package, config):
+    # Imports
+    from configparser import ConfigParser
+    from configparser import ExtendedInterpolation
+    import modules.cross_platform as cp
+    from urllib import request
+    import subprocess
+    import shutil
+
+    # Config
+    os_platform = config['OS.platform']
+    remote_location = config['Remote.location']
+    remote_branch = config['Remote.branch']
+    file_extension = config['Remote.file_extension']
+
+    # Downloading
+    print('--> Downloading Package File')
+    file_name = package + file_extension
+    file_url = remote_location + os_platform + '/' + remote_branch + '/new-scripts/' + file_name
+    cp.chdir('scripts/', os_platform)
+    with request.urlopen(file_url) as response, open(
+        file_name, 'wb') as  package_file:
+            shutil.copyfileobj(response, package_file)
+
+    # Additional Resources
+    print('--> Downloading Additional Resources')
+    package_file = ConfigParser(interpolation=ExtendedInterpolation())
+    package_file.read(file_name)
+    package_download = package_file['Script']['download']
+    with open(package + '.log', 'a') as package_log:
```

Comment:
I have nothing against log files, but why do you think it might be useful? Maybe when users have some troubles we need to get some infos and the log file can help, am i wrong?

Reply:
I am outputting to log files because this allows only a few lines for output of the step and allows us to debug, but not cause clutter in the terminal for other users.

Reply:
I totally agree with you